src/main/java/io/aiven/kafka/connect/gcs/GcsSinkConfig.java

```diff
@@ -275,77 +275,6 @@ public void ensureValid(final String name, final Object value) {
 
     }
 
-    private static void addFormatConfigGroup(final ConfigDef configDef) {
```

Comment:
This is duplicate code, we have this impl in commons right?

Reply:
Exactly.